help.py

```diff
@@ -107,11 +107,30 @@ def get_grid_latlon(dc, lat, lon):
     return argmax_nd((dc.lat == lat) * (dc.lon == lon))
 
 
+def latlon_sel(dc, lat, lon, return_indices = False):
+    """
+    Find the closest grid cell of xr.DataArray dc for a given lat and lon in case of a non-orthogonal grid (when the .sel(lat = ..., lon = ...) functionality won't work)
+    """
+
+    flat_lats = dc.lat.data.flatten()
+    flat_lons = dc.lon.data.flatten()
+
+    distances = np.sqrt((flat_lats - lat)**2 + (flat_lons - lon)**2)
```

Comment:
Certainly efficient but increasing error with increasing distance to the equator. Using GDAL's osr and ogr modules, one can create geographic coordinates (points), assign them to a specific coordinate reference system (CRS), and compute the distance.

Reply:
I flagged this as a TO DO for a future PR!